src/providers/local/exit-pool.provider.ts

```diff
@@ -397,7 +400,9 @@ const provider = (props: Props) => {
         slippageBsp: slippageBsp.value,
         tokenInfo: exitTokenInfo.value,
         prices: prices.value,
-        relayerSignature: '',
+        approvalActions: approvalActions.value,
+        bptInValid: bptInValid.value,
```

Comment:
Looking at the use if this value `bptInValid` I don't think it's relevant. Is it ever invalid? How can the slider be in an invalid state?

Reply:
Slider can't go over account balance, but manually typing the input can.